### PR TITLE
fix the stochastic collocation with gamma distribution 

### LINKED ISSUE #693: gPC ROM generated using SparseGridCollocation with Gamma distribution is not working correctly
--------
Issue Description
--------
##### What did you expect to see happen?
The gPC ROM generated using SparseGridCollocation with Gamma distribution should generate similar results with MonteCarlo

##### What did you see instead?
The predicted value from gPC does not agree with the Code executions, the inputs are youngsModulus, and force. The outputs are disp_x, and disp_y. 
Results from Code:
```
youngsModulus,force,disp_x,disp_y
176574.579712,-132.140792122,0.0254384584278,-0.172387649733
425755.525175,-7.85914919469,0.000627476268902,-0.0042521900282
190033.125005,-83.1397770671,0.0148717340472,-0.100780606935
210154.666871,-14.6123587696,0.00236354215596,-0.0160169091405
231284.205188,-43.5948157121,0.00640723917344,-0.0434196476773
267340.506387,-266.852951695,0.0339304257776,-0.229934780476
```
Results from gPC:
```
youngsModulus,force,disp_x,disp_y
176574.579712,-132.140792122,5.47380623359e-09,-3.70940949265e-08
425755.525175,-7.85914919469,1.40689665974e-10,-9.53405290966e-10
190033.125005,-83.1397770671,3.17405035702e-09,-2.15094433782e-08
210154.666871,-14.6123587696,4.97911911674e-10,-3.37417711342e-09
231284.205188,-43.5948157121,1.33521875082e-09,-9.04831646883e-09
267340.506387,-266.852951695,7.05075070558e-09,-4.7780503148e-08
```

If I change the Gamma distribution to Normal or using NDspline ROM with Monte Carlo, both will generate similar results compared to the code outputs. 
##### Do you have a suggested fix for the development team?
I guss the problem is scaling/normalization factors for the polynomial coefficients. 

##### Please attach the input file(s) that generate this error.  The simpler the input, the faster we can find the issue.

[tests.zip](https://github.com/idaholab/raven/files/2202323/tests.zip)

----------------
For Change Control Board: Issue Review
----------------
This review should occur before any development is performed as a response to this issue.
- [x] 1. Is it tagged with a type: defect or improvement?
- [x] 2. Is it tagged with a priority: critical, normal or minor?
- [x] 3. If it will impact requirements or requirements tests, is it tagged with requirements?
- [x] 4. If it is a defect, can it cause wrong results for users? If so an email needs to be sent to the users.
- [x] 5. Is a rationale provided? (Such as explaining why the improvement is needed or why current code is wrong.)

-------
For Change Control Board: Issue Closure
-------
This review should occur when the issue is imminently going to be closed.
- [x] 1. If the issue is a defect, is the defect fixed?
- [x] 2. If the issue is a defect, is the defect tested for in the regression test system?  (If not explain why not.)
- [x] 3. If the issue can impact users, has an email to the users group been written (the email should specify if the defect impacts stable or master)?
- [x] 4. If the issue is a defect, does it impact the latest stable branch? If yes, is there any issue tagged with stable (create if needed)?
- [x] 5. If the issue is being closed without a merge request, has an explanation of why it is being closed been provided?

### PR DESCRIPTION
**Pull request by wangcj05**
--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
close #693 

##### What are the significant changes in functionality due to this change request?
bug fix for #693 

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

File: tests/framework/Samplers/SparseGrid/test_scgpc_gamma_large_scale.xml [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  </RunInfo>
+  <TestInfo>
+    <name>framework/Samplers/SparseGrid/gamma_large_scale</name>
+    <author>wangc</author>
+    <created>2018-07-26</created>
+    <classesTested>Samplers.SparseGridCollocation</classesTested>
+    <description>
+        This tests using SparseGridCollocation with gamma-distributed inputs with large scale parameters
+        to test the fix for issue #693
+    </description>
```
Comment (PaulTalbot-INL) [inline tests/framework/Samplers/SparseGrid/test_scgpc_gamma_large_scale.xml]: Is this test not analytic?

**Comment by wangcj05** [comment]
@PaulTalbot-INL I created an analytic test for this PR.

**Comment by PaulTalbot-INL** [comment]
Did you add the `<analytic>` node to the corresponding test? I didn't see it in the commit.

**Comment by PaulTalbot-INL** [comment]
This is ready to merge except for the drafted email to the user group.

**Comment by alfoa** [comment]
Checklist passed...

@wangcj05  Please email the users...

Merging...

**Pull request merged**